Modules/Cloud/classes/class.ilCloudPluginCreateFolderGUI.php

```diff
@@ -99,10 +99,10 @@ public function createFolder()
             $response->folder_id = $new_node->getId();
             $response->folder_path = $new_node->getPath();
             $response->success = true;
-            $response->message = $tpl->getMessageHTML($lng->txt("cld_folder_created"), "success");
+            $response->message = ilUtil::getSystemMessageHTML($lng->txt("cld_folder_created"), "success");
```

Comment:
Why did you move to static approach here?

Reply:
Because I do not see this method to be there for a very long time and thus did not want to invest more effort into it. As the dicto-rule says, I see this method could be replace now already by using UI-Message-Box plus -Renderer. A better solution would be (as you also suggest in the comment below) that at some point some service takes care about these messages and the rendering of them.

Reply:
OK, this is fine.